### PR TITLE
Update lamigo/lpurge for lipe compatibililty

### PR DESCRIPTION
**Pull request by utopiabound**
Remove ltuer as it's unused.

Signed-off-by: Nathaniel Clark <nclark@whamcloud.com>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/2292)

### CODE REVIEW

**Review by ip1981 — APPROVED** [review]

File: iml-agent/src/action_plugins/lamigo.rs [hunk 1]
```diff
@@ -168,31 +149,13 @@ mod lamigo_tests {
...
         };
 
         assert_eq!(
             format_lamigo_conf_file(&config, "/etc/lamigo/{fs}-{mdt}.conf").unwrap(),
             PathBuf::from("/etc/lamigo/LU_TEST1-MDT0010.conf")
         )
     }
-
-    #[tokio::test]
-    async fn test_works() {
```
Comment (mkpankov) [inline iml-agent/src/action_plugins/lamigo.rs:181]: Why remove the test?

Comment (utopiabound) [reply iml-agent/src/action_plugins/lamigo.rs:181]: because we no longer generate lamigo service files.

**Review by mkpankov — DISMISSED** [review]

**Review by nlinker — DISMISSED** [review]

File: iml-agent/src/action_plugins/action_plugin.rs [hunk 2]
```diff
@@ -67,15 +67,15 @@ pub fn create_registry() -> action_plugins::Actions {
...
-        .add_plugin("create_lamigo_service", lamigo::create_lamigo_service_unit)
         .add_plugin(
             "configure_ntp",
             action_configure::update_and_write_new_config,
         )
         .add_plugin("is_ntp_configured", is_ntp_configured::is_ntp_configured)
-        .add_plugin("create_ltuer_conf", ltuer::create_ltuer_conf)
         .add_plugin("create_ldev_conf", ldev::create)
+        // HotPools
+        .add_plugin("config_lpurge", lpurge::create_lpurge_conf)
```
Comment (jgrund) [inline iml-agent/src/action_plugins/action_plugin.rs:77]: We should have the plugin names match the action plugin fns.

**Review by mkpankov — APPROVED** [review]

File: iml-agent/src/action_plugins/lamigo.rs [hunk 3]
```diff
@@ -34,24 +34,33 @@ pub struct Config {
...
-    mailbox: String,
+    /// IML mailbox name, e.g. `mailbox1`, where `lamigo` will write FIDs for mirror extend
+    mailbox_extend: String,
+
+    #[structopt(long)]
+    /// IML mailbox name, e.g. `mailbox2`, where `lamigo` will write FIDs for mirror resync
+    mailbox_resync: String,
+
+    #[structopt(long)]
+    heatfn: Option<u32>,
```
Comment (jgrund) [inline iml-agent/src/action_plugins/lamigo.rs:49]: Would be good to document what `heatfn` is

**Review by jgrund — APPROVED** [review]

**Pull request merged**